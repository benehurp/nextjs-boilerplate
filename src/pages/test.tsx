import { render, screen } from '@testing-library/react'

import Home from '.'

describe('<Home />', () => {
   it('shold render the heading', () => {
      render(<Home />)

      expect(screen.getByRole('heading', { name: /home/i })).toBeInTheDocument()
   })
})
