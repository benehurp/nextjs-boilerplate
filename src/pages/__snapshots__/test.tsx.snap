// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> shold render the heading 1`] = `
<h1>
  Home
</h1>
`
